pkg/binding/transport.go

```diff
@@ -27,11 +28,15 @@ func NewTransportAdapter(s Sender, r Receiver, senderContextDecorators []func(co
 	return &BindingTransport{Sender: s, Receiver: r, SenderContextDecorators: senderContextDecorators}
 }
 
-func (t *BindingTransport) Send(ctx context.Context, e event.Event) (context.Context, *event.Event, error) {
+func (t *BindingTransport) Send(ctx context.Context, e event.Event) error {
 	for _, f := range t.SenderContextDecorators {
 		ctx = f(ctx)
 	}
-	return ctx, nil, t.Sender.Send(ctx, EventMessage(e))
+	return t.Sender.Send(ctx, EventMessage(e))
+}
+
+func (t *BindingTransport) Request(ctx context.Context, e event.Event) (*event.Event, error) {
+	return nil, errors.New("Transport.Request is not yet supported by binding transport.")
```

Comment:
If you meant to use this func as default impl for this method, maybe rename it to `Transport.Request is not supported by this transport.`

Reply:
In a followup I am implementing this. The error is the detail we want for now to find where it is not implemented. All the transport parts will be removed anyway.